ir-engine/etherealengine-archive
# IR-1820 playmode interactables

## Summary
Allowing interactable usage in studio editor playmode (includes mountpoint, grabbables, link)

## Subtasks Checklist

## Breaking Changes

## References
[IR-1820](https://tsu.atlassian.net/browse/IR-1820)

## QA Steps
Enter studio, open a scene (ensure scene has interactables in it), hit the play button on the top bar to spawn an avatar and preview.

You should be able to interact with MountPointComponents, vanilla InteractableComponents, and LinkComponents

NOTES: 
- GrabbableComponents will work once their specific system is fixed
- A bug in play mode prevents exiting and re-entering play mode. you will need to refresh your page if you exit play mode before you can re-enter it

[IR-1820]: https://tsu.atlassian.net/browse/IR-1820?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
